app/views/home/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav>
```

Comment:
@PeregrineReed Looks good! Since we will need a partial for the navigation in a bit, is it ok if I create that after we merge this branch?